From 554b8c274f09928af7881e6a41dbce0c10d7dc7d Mon Sep 17 00:00:00 2001
From: Shu Muto <shu-mutou@rf.jp.nec.com>
Date: Thu, 15 Jun 2017 18:06:47 +0900
Subject: [PATCH 19/29] Enable to set protocols of WebSocket for serial console

This patch enables to set protocols of WebSocket for serial console.
Also, this enables to use serial console not only from instance
but also plugins.
e.g. Zun UI can access container console provided by docker.

Change-Id: Ib0df9ddfc74f98bfea75abce3b5d5479e3cd47bd
Needed-By: I41f236f2762613e56748bb05eb7ce0e4c26158d2
Closes-Bug: #1698092
---
 openstack_dashboard/templates/serial_console.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openstack_dashboard/templates/serial_console.html b/openstack_dashboard/templates/serial_console.html
index 0ff0c06..d1434e7 100644
--- a/openstack_dashboard/templates/serial_console.html
+++ b/openstack_dashboard/templates/serial_console.html
@@ -4,7 +4,7 @@
 <head>
   <meta content='IE=edge' http-equiv='X-UA-Compatible' />
   <meta content='text/html; charset=utf-8' http-equiv='Content-Type' />
-  <title>{{instance_name}} ({{instance_id}})</title>
+  <title>{{page_title}}</title>
   {% load compress %}
   {% compress css %}
   <link rel="stylesheet" href="{{ STATIC_URL }}dashboard/scss/serial_console.scss" type="text/scss" media="screen" />
-- 
1.8.3.1

